MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+include versioneer.py
+include lib/nc_time_axis/_version.py
```

Comment:
Don't use `.pth` files... (`python setup.py develop` or `pip install --editable .` work a treat) 😉 

There is limited impact to the setup being on the path in those situations.